src/cascade/input_data/configuration/construct_study.py

```diff
@@ -0,0 +1,118 @@
+"""
+Given data from the db, organize it into columns of data which then go
+onto the model.
+"""
+from collections.__init__ import defaultdict
+
+import numpy as np
+import pandas as pd
+
+from cascade.input_data import InputDataError
+from cascade.input_data.configuration.covariate_records import CovariateRecords
+from cascade.input_data.db.study_covariates import _get_study_covariates, \
+    covariate_ids_to_names
+from cascade.core.log import getLoggers
```

Comment:
Above in construct_country.py, we have this:  from cascade.core import getLoggers  
rather than: from cascade.core.log

Reply:
Yeah, I let that slip. The logging is pulled up to core.